components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -1284,18 +1284,19 @@ private String extractPropertyName(String pathProp) {
     @Deprecated
     public UserStoreAttributeMapping getUserStoreMappingAttributes(String typeId) {
 
-        return getUserStoreMappingAttributes(typeId, false);
+        return getUserStoreMappingAttributes(typeId, true);
```

Comment:
`getUserStoreMappingAttributes(String typeId)` this method doesn't seem to be having any usages and it was introduced fairly recently(not in IS 5.11.0 release). Shall we check remove the method without deprecating it

Reply:
fixed with 893bbd644c7284d2eccb04b1c29a9f7686654fa8